Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/MainActivityViewModel.kt

```diff
@@ -145,16 +150,18 @@ class MainActivityViewModel @AssistedInject constructor(
 
     private suspend fun QrCodeExtractor<CoronaTestQRCode>.handleCoronaTestQr(uriString: String) = try {
         val qrCode = extract(rawString = uriString)
-        val test = submissionRepository.testForType(qrCode.type).first()
-        when {
-            test != null -> event.postValue(MainActivityEvent.GoToDeletionScreen(qrCode))
-            else -> event.postValue(MainActivityEvent.GoToSubmissionConsentFragment(qrCode))
```

Comment:
you need to remove those events from `MainActivityEvent`

Reply:
done